### ISSUE TITLE
More than 10 values in the minutes field results in incorrect frequencies

### ISSUE THREAD

**Issue by CMHanna** [bunkat/later#100]
For example this cron 4,9,14,19,24,29,34,39,44,49,54,59 * * * *
```javascript
var freq = later.parse.cron('4,9,14,19,24,29,34,39,44,49,54,59 * * * *', false);
var now = new Date(2015, 4, 11);
var next = later.schedule(freq).next(12, now);
```
Results in the following:
```
Expected Date(Mon May 11 2015 00:39:00 GMT-0400 (Eastern Daylight Time)) to equal Date(Mon May 11 2015 00:04:00 GMT-0400 (Eastern Daylight Time)).
Expected Date(Mon May 11 2015 00:44:00 GMT-0400 (Eastern Daylight Time)) to equal Date(Mon May 11 2015 00:09:00 GMT-0400 (Eastern Daylight Time)).
Expected Date(Mon May 11 2015 00:49:00 GMT-0400 (Eastern Daylight Time)) to equal Date(Mon May 11 2015 00:14:00 GMT-0400 (Eastern Daylight Time)).
Expected Date(Mon May 11 2015 00:54:00 GMT-0400 (Eastern Daylight Time)) to equal Date(Mon May 11 2015 00:19:00 GMT-0400 (Eastern Daylight Time)).
Expected Date(Mon May 11 2015 00:59:00 GMT-0400 (Eastern Daylight Time)) to equal Date(Mon May 11 2015 00:24:00 GMT-0400 (Eastern Daylight Time)).
Expected Date(Mon May 11 2015 01:39:00 GMT-0400 (Eastern Daylight Time)) to equal Date(Mon May 11 2015 00:29:00 GMT-0400 (Eastern Daylight Time)).
Expected Date(Mon May 11 2015 01:44:00 GMT-0400 (Eastern Daylight Time)) to equal Date(Mon May 11 2015 00:34:00 GMT-0400 (Eastern Daylight Time)).
Expected Date(Mon May 11 2015 01:49:00 GMT-0400 (Eastern Daylight Time)) to equal Date(Mon May 11 2015 00:39:00 GMT-0400 (Eastern Daylight Time)).
Expected Date(Mon May 11 2015 01:54:00 GMT-0400 (Eastern Daylight Time)) to equal Date(Mon May 11 2015 00:44:00 GMT-0400 (Eastern Daylight Time)).
Expected Date(Mon May 11 2015 01:59:00 GMT-0400 (Eastern Daylight Time)) to equal Date(Mon May 11 2015 00:49:00 GMT-0400 (Eastern Daylight Time)).
Expected Date(Mon May 11 2015 02:39:00 GMT-0400 (Eastern Daylight Time)) to equal Date(Mon May 11 2015 00:54:00 GMT-0400 (Eastern Daylight Time)).
Expected Date(Mon May 11 2015 02:44:00 GMT-0400 (Eastern Daylight Time)) to equal Date(Mon May 11 2015 00:59:00 GMT-0400 (Eastern Daylight Time)).
```

Similarly this also fails:
```
2,4,6,8,10,12,14,16,18,20,22 * * * *
```

and these work as expected:
```
2,4,6,8,10,12,14,16,18,20 * * * *
4,9,14,19,24,29,34,39,44,49 * * * *
```
Have not tried if similar behavior occurs with more than 10 values for hours, dayOfMonth and month fields.

**Comment by bunkat** [comment]
What does `console.log(freq)` produce?

**Comment by CMHanna** [comment]
![capture](https://cloud.githubusercontent.com/assets/945026/7569984/95e68e96-f7da-11e4-9de5-9eb7edee7ed4.JPG)

**Comment by bunkat** [comment]
Hmm... looks like the array values are being sorted improperly.  The values must be sorted before schedules can be calculated.  I took a quick look but it wasn't obvious what is causing this (and why it wouldn't always be a problem).  In the cron parser, seems like the problem is in the `itemSorter` (cron.js line 225) function but that shouldn't change the order except for hashes.

**Comment by bunkat** [comment]
Fixed in v1.1.8.
